components/AddForm/AddForm.jsx

```diff
@@ -1,70 +1,48 @@
 import { useState } from 'react';
 import PropTypes from 'prop-types';
-import cx from 'classnames';
+import { useForm } from 'react-hook-form';
 
 import Button from 'components/Button/Button';
 import { useAuth } from 'components/UserContext/UserContext';
-import { Select } from 'components/Form';
+import { Autocomplete } from 'components/Form';
 import { populateChildForm } from 'utils/populate';
-import ADULT_CATE from 'data/googleForms/adultCategories';
 import ADULT_FORMS from 'data/googleForms/adultForms';
-import CHILD_CATE from 'data/googleForms/childCategories';
 import CHILD_FORMS from 'data/googleForms/childForms';
 
 const AddForm = ({ person }) => {
   const { user } = useAuth();
   const [url, setUrl] = useState();
-  const [categoryValue, setCategoryValue] = useState();
   const ageContext = person && person.ageContext;
-  const category = ageContext === 'C' ? CHILD_CATE : ADULT_CATE;
   const forms = ageContext === 'C' ? CHILD_FORMS : ADULT_FORMS;
-  const filteredForms = forms.filter((form) => form.category === categoryValue);
-  const isOneFormOnly = filteredForms.length === 1;
-  const formURL = isOneFormOnly ? filteredForms[0].value : url;
+  const { control } = useForm({ defaultValues: '' });
 
   return (
     <>
-      <div className={cx('govuk-grid-row')}>
-        <Select
-          govGrid="one-half"
-          name="category"
-          options={category}
-          label="Choose a form category"
-          placeHolder="Choose one"
-          width={30}
-          onChange={(value) => setCategoryValue(value)}
+      <div className="govuk-form-group">
+        <Autocomplete
+          name="formList"
+          options={forms}
+          label="Choose a form"
+          control={control}
```

Comment:
shouldn't we pass the control only if we are using `useForm`?

Reply:
We get a runtime error "controller is missing control" without it

Reply:
It's because you should wrap `AutocompleteComponents` with `Controller` only if it's meaningful. In this case we are not using `useForm` so it doesn't make sense to be wrapper in `Controller`.

Reply:
see comment above